app/src/app/api/v0/assistants/route.ts

```diff
@@ -0,0 +1,31 @@
+import { apiHandler } from "@/util/api";
+import { openai } from "@/util/openai";
+import { NextResponse } from "next/server";
+
+// Create a new assistant
+export const POST = apiHandler(async () => {
+  // Currently we create the assistant via OpenAI web interface and do not implement this function here
+  return NextResponse.json({ error: "Method Not Allowed", status: 405 });
+  // TODO: should be an admin function
+  const userID = "id_123456"; // TODO: Get userID
```

Comment:
We could get the logged in user id from the session here.

Reply:
I added `const userID = session?.user.id;`